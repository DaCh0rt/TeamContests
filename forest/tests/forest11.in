232 707
-6155 -1862
5803 3139
-4607 6263
-9397 2987
-1135 -5851
-2931 -4873
7291 6440
-3872 -5249
-6598 8958
8829 -1785
-9473 8379
1079 -5463
-3677 6836
8369 -8263
2118 5456
3409 8775
1451 4718
8281 3622
-7622 -1859
292 9706
-2348 1576
-616 4537
4779 7243
8542 -4902
-7410 6542
1235 -8746
4211 402
-7003 -9805
9544 9265
-5007 -559
9227 -1810
8252 -9230
-3427 7732
-5049 -8615
4191 3177
-4780 -2110
-4461 -8605
-3384 9517
-2669 4852
-6162 -4888
-1810 8888
-2259 -9911
-1695 -7637
5945 -8025
-7406 2779
8777 -1832
-2209 2980
-2652 -3562
3275 -7331
7347 2080
-1390 8916
-9889 -6545
-5559 -673
5766 -5455
-5100 6079
9752 -6736
-5828 1555
-5927 7049
-1971 -4565
1946 -7583
7108 -6161
3306 -6536
7841 9518
4053 2329
-391 -9958
3099 8365
8135 7975
-6538 3424
-7611 -5511
88 6821
2783 2104
4982 775
9051 6457
-1205 1755
-1424 -3415
-4319 9668
8490 7503
-7786 -7474
4173 -7257
-2238 1179
-2621 -6568
5684 -7105
3309 688
6340 3137
-9174 -9979
6014 -5639
-1290 -7039
216 -8632
6402 -721
9767 -515
2323 -292
-7068 -9686
6700 -9537
-2204 6433
7111 9158
887 -5448
-1146 -9916
840 -5061
6630 -6921
-6760 7544
443 -9996
-2129 -3135
-8490 -2805
4468 -4784
1852 1514
-3795 -3860
4706 -3339
-319 -2144
9047 -5638
3281 1577
9173 -889
-3093 -5653
-4348 -6221
6331 5555
2112 1460
9500 7217
-6223 -5937
8269 3817
1825 5578
-1901 597
-1649 5202
9011 -7835
2270 -428
9267 -8979
3933 -6448
9692 3008
-6462 -1098
6906 9348
-4321 -6562
4189 600
-2655 2346
1916 4503
-6659 -609
-7975 -7033
1518 -2127
-3783 -6793
7767 1882
8369 2208
2187 6554
4515 -837
7305 7523
-7711 4835
5331 5484
-3828 1380
-3016 -1907
1482 2557
-7140 6563
9954 8030
8843 4386
2430 -7294
-3327 7899
4761 -9212
1034 -3269
501 -2648
2569 -9975
5662 8767
3584 4240
8140 -4425
5812 6612
4810 9162
-901 9075
3255 -6411
5267 1707
-1147 7037
2026 2958
8600 -5227
5104 1131
-7051 1727
6259 3238
-4385 8330
-111 -5694
1237 -689
-9641 2614
4984 8905
-6285 7358
-7578 -1261
-4462 -449
91 -4160
-891 -8786
-2451 -6597
6364 -7803
2430 7611
-3098 -5138
-4781 -4203
4462 9692
70 1237
-2793 1511
7880 -6880
-6380 -342
-2735 -4815
-8284 8710
8279 4050
4157 -5472
9108 5831
-8311 2290
-8698 2469
1466 3743
-1250 -1073
75 -8252
39 2813
2504 -5847
2000 7990
4999 -3273
-5904 152
4520 1336
7602 7388
-8122 -1497
-1743 -4287
972 2078
-9002 7018
-3253 -6153
-8191 8040
5658 5429
-9819 -1899
-498 -1631
5803 -5646
3592 -7483
-8170 5729
-6608 4808
-1782 -6710
1655 9002
1046 -5280
9192 -9358
-6892 -8239
-2258 4607
-9184 1574
8860 8438
-8265 -5065
-6497 5793
-3033 257
-3735 -9220
-9315 -5446
