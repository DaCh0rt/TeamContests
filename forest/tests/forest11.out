79224.45 430549888.10
